### PR TITLE
2. Create acceptStudent function

### LINKED ISSUE #3: Create function to accept a student name acceptStudent, that accepts (ID:string, FullName:string, Age:int, Address:string)
- The status of the student is set to "Applying."

### PR DESCRIPTION
**Pull request by amsolaiman**
- The acceptStudent function is created.
- All necessary value is pre-determined and stored as an example.
- Data values are stored as JSON object.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,20 @@
...
+    return level(db, { valueEncoding: 'json' });
+}
+
+var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
+
+acceptStudent('201812485', 'Abdul Moiz Solaiman', 22, 'Marawi City');
+
+function acceptStudent(id, fullName, age, address){
+    let attributes = { ID:id, Name: fullName, Age: age, Address: address };
+    db.put(id, attributes, function(err){
```
Comment (fencim) [inline main.js]: you don't need to use the promise base approach for the functions minimizing complexity...

**Review by fencim — CHANGES_REQUESTED** [review]
use promise approach using async await..

File: main.js (hunk 1 above)
Comment (amsolaiman) [reply main.js]: it has been updated sir.

File: main.js [hunk 2]
```diff
@@ -1 +1,18 @@
...
+
+acceptStudent('201812485', 'Abdul Moiz Solaiman', 22, 'Marawi City');
+
+function acceptStudent(id, fullName, age, address){
+    let attributes = { ID:id, Name: fullName, Age: age, Address: address };
+    db.put(id, attributes);
+    db.get(id, function(err, value){
+        console.log(value, status[0]);
+    })
+}
```
Comment (lucman-abdulrachman) [inline main.js]: you may want to make use of async /await like
```suggestion
async function acceptStudent(id, fullName, age, address){
    let attributes = { ID:id, Name: fullName, Age: age, Address: address };
    await db.put(id, attributes);
    return await db.get(id)
}
```

File: main.js [hunk 3]
```diff
@@ -1 +1,18 @@
...
+const level = require('level');
+const db = connectToDatabase('./studentdb');
+
+function connectToDatabase(db){
+    return level(db, { valueEncoding: 'json' });
+}
+
+var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
+
+acceptStudent('201812485', 'Abdul Moiz Solaiman', 22, 'Marawi City');
```
Comment (lucman-abdulrachman) [inline main.js]: in preparation for next task, make acceptStudent function as async function , then wrap this line in a function like
```suggestion
(async function() {
     await acceptStudent('201812485', 'Abdul Moiz Solaiman', 22, 'Marawi City');
}());```

**Review by lucman-abdulrachman — CHANGES_REQUESTED** [review]
see suggestions

File: main.js (hunk 2 above)
Comment (amsolaiman) [reply main.js]: the function is updated.

File: main.js (hunk 3 above)
Comment (amsolaiman) [reply main.js]: noted sir.

**Review by fencim — APPROVED** [review]

**Pull request merged**